config/config.example.yml

```diff
@@ -64,13 +64,15 @@ gravatar_default: identicon
 
 # Setup your deploy options for capistrano.
 deployment:
-  hosts:
-    web: errbit.example.com
-    app: errbit.example.com
-    db: errbit.example.com
-  repository: https://github.com/errbit/errbit.git
+  servers:
+    errbit.example.com:
+      roles:
+        - app
+        - db
+        - web
+      user: deploy
```

Comment:
I removed the deployment section from config.yml. Having deployment configuration on config.yml is confusing since config.yml file should be for app configuration.